dartvlc/CMakeLists.txt

```diff
@@ -8,39 +8,56 @@ endif()
 set(LIBRARY_NAME dart_vlc_core)
 project(${LIBRARY_NAME} LANGUAGES C CXX)
 
+set(EXTERNAL_PACKAGE_DIR "${CMAKE_BINARY_DIR}/${LIBRARY_NAME}_packages")
+
 set(LIBVLC_VERSION "3.0.9.2")
-set(LIBVLC_BINARIES "${CMAKE_CURRENT_SOURCE_DIR}/../bin")
+set(DART_VLC_BINARIES "${CMAKE_CURRENT_SOURCE_DIR}/../bin")
+
+set(LIBVLC_ARCHIVE "${DART_VLC_BINARIES}/vlc-${LIBVLC_VERSION}.7z")
+set(LIBVLCPP_ARCHIVE "${DART_VLC_BINARIES}/libvlcpp.zip")
+set(LIBVLC_SOURCE "${EXTERNAL_PACKAGE_DIR}/vlc-${LIBVLC_VERSION}")
+set(LIBVLCPP_SOURCE "${EXTERNAL_PACKAGE_DIR}/libvlcpp-master")
 
-set(LIBVLC_ARCHIVE "${LIBVLC_BINARIES}/vlc-${LIBVLC_VERSION}.7z")
-set(LIBVLCPP_ARCHIVE "${LIBVLC_BINARIES}/libvlcpp.zip")
+set(DARTAPI_ARCHIVE "${DART_VLC_BINARIES}/dart-api.zip")
```

Comment:
I don't see the point of adding this as an archive.
It's just ~5 files as far as I remember. Adding binaries will just unnecessarily complicate change tracking.
Same for libvlcpp (which is quite tiny as well).

If you don't want to add libvlcpp and the dart headers as submodules, let's at least add those files directly as opposed to have CMake unpack them first.